types/artillery/artillery-tests.ts

```diff
@@ -0,0 +1,10 @@
+import { ScenarioContext, RequestResponse, ResponseRequest, Next } from 'artillery';
+
+function test(scenarioContext: ScenarioContext, req: RequestResponse, rep: ResponseRequest, next: Next) {
```

Comment:
Could you link to documentation that shows a test working like this? Does "artillery" not expose any values to be imported?

Reply:
Artillery has before and after hooks that you can use to tie into, however, you tie into these in yml files so I didnt see the need to expose them as a function here since its unlikely you will call them as functions from typescript. See https://artillery.io/docs/http-reference/#advanced-writing-custom-logic-in-javascript

I wasn't quite sure how to write the test for this, any pointers would be great. I had a look through some of the other projects but there doesnt seem to be a standard so I thought just declaring the variables would be enough (to make sure they compile properly). I can change it to just declare them though and get rid of the `test` function since that will be doing the same thing.

Reply:
Eh, it's good enough as is.